There is a test failure about JSON response too:

```
>       self.assertEqual(set(resp.json().items()[0][1][0].keys()),
                         {'state', 'count', 'color'})
E       TypeError: 'dict' object is not callable
```

--- a/UPDATING.md
+++ b/UPDATING.md
@@ -1212,7 +1212,6 @@ Now the `dag_id` will not appear repeated in the payload, and the response forma
 }
 ```
 
-
 ## Airflow 1.10.10
 
 ### Setting Empty string to a Airflow Variable will return an empty string

--- a/UPDATING.md
+++ b/UPDATING.md
@@ -1164,6 +1164,55 @@ The goal of this change is to achieve a more consistent and configurale cascadin
 
 When initializing a Snowflake hook or operator, the value used for `snowflake_conn_id` was always `snowflake_conn_id`, regardless of whether or not you specified a value for it. The default `snowflake_conn_id` value is now switched to `snowflake_default` for consistency and will be properly overriden when specified.
 
+### Simplify the response payload of endpoints /dag_stats and /task_stats
+
+The response of endpoints `/dag_stats` and `/task_stats` help UI fetch brief statistics about DAGs and Tasks. The format was like
+
+```json
+{
+    "example_http_operator": [
+        {
+            "state": "success",
+            "count": 0,
+            "dag_id": "example_http_operator",
+            "color": "green"
+        },
+        {
+            "state": "running",
+            "count": 0,
+            "dag_id": "example_http_operator",
+            "color": "lime"
+        },
+        ...
+],
+...
+}
+```
+
+The `dag_id` was repeated in the payload, which makes the response payload unnecessarily bigger.
+
+Now the `dag_id` will not appear repeated in the payload, and the response format is like
+
+```json
+{
+    "example_http_operator": [
+        {
+            "state": "success",
+            "count": 0,
+            "color": "green"
+        },
+        {
+            "state": "running",
+            "count": 0,
+            "color": "lime"
+        },
+        ...
+],
+...
+}
+```
+
+
 ## Airflow 1.10.10
 
 ### Setting Empty string to a Airflow Variable will return an empty string

--- a/tests/www/test_views.py
+++ b/tests/www/test_views.py
@@ -521,6 +521,8 @@ class TestAirflowBaseViews(TestBase):
     def test_task_stats(self):
         resp = self.client.post('task_stats', follow_redirects=True)
         self.assertEqual(resp.status_code, 200)
+        self.assertEqual(set(resp.json().items()[0][1][0].keys()),
+                         {'state', 'count', 'color'})
 
     def test_task_stats_only_noncompleted(self):
         conf.set("webserver", "show_recent_stats_for_completed_runs", "False")
@@ -1616,6 +1618,8 @@ class TestDagACLView(TestBase):
         self.login()
         resp = self.client.post('dag_stats', follow_redirects=True)
         self.check_content_in_response('example_bash_operator', resp)
+        self.assertEqual(set(resp.json().items()[0][1][0].keys()),
+                         {'state', 'count', 'color'})
 
     def test_dag_stats_failure(self):
         self.logout()

